CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 06 (11 January 2017)
+
+* Initial SQL and query banding support (#123)
+* Fixed bug in xfiles to only attempt to parse tde or tds files (#117, #118)
```

Comment:
I'd rephrase to "Allow opening workbooks with externalfilecache entries" since that's the type of workbook that caused the bug